apollo-ios-codegen/Sources/ApolloCodegenLib/Templates/EnumTemplate.swift

```diff
@@ -43,15 +44,16 @@ struct EnumTemplate: TemplateRenderer {
       \(if: shouldRenderDocumentation, "///")
       \(documentation: "**Deprecated**: \($0.escapedSwiftStringSpecialCharacters())")
       """ })
+    \(graphqlEnumValue.name.typeNameDocumentation) 
```

Comment:
Haha this was a silly one.

There was an extra whitespace character at the end of this line. The `TemplateString` removes the line only if the line is empty, and since it wasn't that's where your issue was coming from.

This should make it work correctly.

```suggestion
    \(graphqlEnumValue.name.typeNameDocumentation)
```